docs/source/en/llm_tutorial_optimization.md

```diff
@@ -662,7 +664,15 @@ Using the key-value cache has two advantages:
 
 > One should *always* make use of the key-value cache as it leads to identical results and a significant speed-up for longer input sequences. Transformers has the key-value cache enabled by default when making use of the text pipeline or the [`generate` method](https://huggingface.co/docs/transformers/main_classes/text_generation).
 
-Note that the key-value cache is especially useful for applications such as chat where multiple passes of auto-regressive decoding are required. Let's look at an example.
+Tip warning={true}>
```

Comment:
Added this warning with the recent finding that using caches produces tiny fluctuations in the model, so users can find this info more easily :)